company/goals/guidelines.md

```diff
@@ -49,6 +44,27 @@ A goal's format is:
    - Write your goals so they communicate effectively to an audience who understands that goals aren't promises. Don't write goals so cautiously that they are hard to understand. (For example, "Improve code intelligence support for Java" is a good goal, especially with well defined outcomes. But "Investigate feasibility of improvements to Java code intelligence" is probably bad.)
 1. If a goal's success criteria can be reduced to a single quantitative metric, that's great. But don't try to force it. It's OK to rely on human judgment to judge the success of a goal.
 
+## Goal formats
+
+A goal's general format is:
+
+> **Goal title** \
+> Description of the goal and how to evaluate whether we met the goal (with a link to an automatically updated metrics dashboard, if appropriate).
+
+### Engineering team goals 
+
+Each engineering team should have its own goals page with explicit goals. Because engineering team goals are larger than a single person's goal, we use the following format: 
+
+> **Goal title:** _The title for a vision that is achievable within a 6-12 month time frame._ 
```

Comment:
Is a vision that is achievable in 3-4 months too small to list as a goal?

Reply:
I don't know that it is, this was just taken from [a description](https://docs.google.com/document/d/1xszELbG-5urpXf-FHPc1tLDMBb_PM3MTFjFSOFRHPTo/edit#bookmark=id.kkwv4akl15r4) @christinaforney gave in a product sync one day. 

I wonder if there are any 3-4 month goals that don't roll up into larger 6-12 month goals, however, and perhaps the implication was that 3-4 month goals are better as milestones in that larger goal?

Reply:
This is fine as is. Upon second reading, I think the "within" existing languages allows it anyway.